services/bots/src/github-webhook/handlers/validate-cla.ts

```diff
@@ -0,0 +1,284 @@
+import { ConfigService } from '@nestjs/config';
+import { BaseWebhookHandler } from './base';
+
+import { EventPayloadMap } from '@octokit/webhooks-types';
+
+import { DynamoDB } from 'aws-sdk';
+
+const ignoredAuthors: Set<string> = new Set([
+  // Ignore bot accounts that are not masked as bots
+  'travis@travis-ci.org',
+  'ImgBotHelp@gmail.com',
+  'support@lokalise.com',
+]);
+
+const ignoredRepositories: Set<string> = new Set([
+  // Ignore repositories that do not contain code
+  'home-assistant/home-assistant.io',
+  'home-assistant/assets',
```

Comment:
Should we add more repos?
Like:
- companion docs
- data portal
- architecture
- .github
- others?

Reply:
yeah, probably only code related ones.